### DISCUSSION TITLE
Ideas from @TomUgulini

### DISCUSSION THREAD

**Discussion by broofa** [broofa/flightcard#8] [General]
@TomUgulini wrote:

----
Thought I'd try a little experiment. To facilitate brainstorming and "stream of consciousness" discussions I will try to capture and organize what I hear members saying into a lightweight design document. I have no particular format in mind, so let me know what works best for the group. I started with a Google spreadsheet, let me know if it's visible to those that are interested [click here to see document](https://docs.google.com/spreadsheets/d/1V_TY0Xkvxyuei5-nu_c7rLtd9ptg7cFgFNLlODs6ESM/edit?usp=sharing)

**Comment by broofa** [comment]
@TomUgulini Thanks for pulling those comments/requirements together.  There's some good thoughts in there I hadn't recorded.

Any chance I can get you to create one or more ["Idea" discussions](https://github.com/broofa/flightcard/discussions/new?category=ideas) instead of using Google Sheets?   It'll be much easier for I and others to respond (and take action on) the various items if that stuff lives inside Github.

**Reply by TomUgulini** [reply]
Absolutely agree - all idea discussions should start in GitHub, I'll transfer mine over from Google Sheets.

I find that overtime if a project has long running discussion threads (not necesarily assuming this one will, but just in case) it can be useful for someone new joining in to get up to speed by reading a summarized "plan of record".  (For example, the current POR for V1 is that onsite internet connectivity is a requirement for all users).  That's why I dumped my thoughts into a Google Sheet, but perhaps the correct home for this would be in the Wiki section of GitHub for this project?

**Comment by spindrift22** [comment]
I am currently flying 2-stage rockets. I usually have motor combinations decided on several weeks prior to launch so have been in the habit of filling out flight cards well in advance of going out to Brothers.  Because of the complexity of a 2-stage the current flight cards don't really offer enough information for the RSO so I bring full design reports. These show changes in cg/cp under different motor combinations. different thrust ratios at takeoff, and the speeds achieved to clear the tower as well as the performance of the sustainer after separation.  I realize I am very much in the minority but wonder if there can be additional room left on the flight card maybe under a miscellaneous category?

**Reply by broofa** [reply]
> additional room left on the flight card maybe under a miscellaneous category?

@spindrift22 I have a "Notes" field in the current UI (see below) that I just sort of tossed in without much thought.  I'm not entirely sure I'll keep this, but there should be something along these lines that allows free-form text to be entered.

![image](https://user-images.githubusercontent.com/164050/117549309-660db000-afd5-11eb-9e6e-742ca21dde6a.png)

**Reply by broofa** [reply]
@spindrift22 FWIW, I've been getting back into the app and have improved the UI for adding motors configuration. It's now possible to specify complex arrangements for both clustering and staging. 
 
I've also added a motor analysis section that provides Thrust:weight ratio and minimum recommended rail travel.

For example, a 3-stage rocket with a 2 x E-motor first stage looks like this:
<img width="728" alt="CleanShot 2022-07-12 at 11 02 27@2x" src="https://user-images.githubusercontent.com/164050/178561897-a6f0c120-2cd2-49aa-9d08-47d8777a30f4.png">

The "Add motor" panel (featuring auto-suggestions for production motors from Thrustcurve database):
<img width="510" alt="CleanShot 2022-07-12 at 11 05 17@2x" src="https://user-images.githubusercontent.com/164050/178562421-63a94876-12e1-454b-885c-347dcde88864.png">

The motor detail panel (pops up when you click the thrustcurve logo by any motor name:
<img width="518" alt="CleanShot 2022-07-12 at 11 05 30@2x" src="https://user-images.githubusercontent.com/164050/178562760-87f6cbe4-05ee-466b-aeb6-5b05c4beddf7.png">
